analysis_options.yaml

```diff
@@ -222,7 +222,6 @@ linter:
     - unnecessary_string_interpolations
     - unnecessary_this
     - unnecessary_to_list_in_spreads
-    - unreachable_from_main
```

Comment:
Please just comment out this rule and add a comment why it should not be enabled yet (see other deactivated rules). Otherwise, this may get re-enabled pretty quickly.

Reply:
Done :)